### DISCUSSION TITLE
Disney Junior East [US]

### DISCUSSION THREAD

**Discussion by dantheman121** [iptv-org/community#450] [Show and tell]
### Channel ID

DisneyJuniorEast.us

### Stream URL (optional)

https://ddh2.cdnhks.lol/ddh2/premium652/tracks-v1a1/mono.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by PopeyeTheSai10r** [comment]
The link don't work.
